composables/useToast.ts

```diff
@@ -0,0 +1,16 @@
+// vue composables api for toast message function
+export default function useToast() {
```

Comment:
It seems that this composables method was not being used anymore. Should we remove it ?